index.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   <meta name="description" content="Insalan">
+  <link rel="stylesheet" type="text/css" href="/src/assets/style.css" media="screen" />
```

Comment:
critical css can go on global (read as non-scoped) <style> tag on App.vue

Reply:
solved in [67b5b74](https://github.com/InsaLan/frontend-insalan.fr/pull/40/commits/67b5b7444523a779cb437504c7abb6ab7f2a3c8d)